sherpa/utils/__init__.py

```diff
@@ -993,24 +993,31 @@ def pad_bounding_box(kernel, mask):
 eps = numpy.finfo(numpy.float32).eps
 
 
-def filter_bins(mins, maxes, axislist):
+def filter_bins(mins, maxes, axislist, integrated=False):
     """What mask represents the given set of filters?
 
+    The ranges are treated as inclusive at both ends if integrated is
+    False, the default, otherwise the lower limit is inclusive but the
+    upper limit is exclusive.
+
     Parameters
     ----------
     mins : sequence of values
        The minimum value of the valid range (elements may be None).
-       When not None, it is treated as an inclusive limit, so
```

Comment:
Like above, I don't understand what that can be a sequence of values. Apparently, I miss something fundamental.

Reply:
I think this is similar to https://github.com/sherpa/sherpa/pull/1216#discussion_r704488780